tests/unit/test_pareto.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2020 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+import tensorflow as tf
+
+from trieste.data import Dataset
+from trieste.utils.pareto import Pareto, non_dominated_sort
+
```

Comment:
could this be in the same style as the tests in the rest of the code? `test_dominated_sort` matches that style, but `TestPareto` uses a different style